tests/wpt/web-platform-tests/html/semantics/forms/the-input-element/month.html

```diff
@@ -13,19 +13,37 @@
   <body>
     <h1>Inputs Month</h1>
     <div style="display: none">
+      <input id="valid_value_1" type="month" value="20133-12" />
+      <input id="valid_value_2" type="month" value="2013-12" />
+      <input id="valid_value_3" type="month" value="0003-01" />
       <input id="valid" type="month" value="2011-11" min="2011-01" max="2011-12" />
       <input id="invalid_value" type="month" value="invalid-month" min="2011-01" max="2011-12"/>
-      <input id="value_can_be_empty_string" type="month" value="2013-06" />
+      <input id="value_can_be_empty_string" type="month" value="" />
```

Comment:
Why is this change necessary?

Reply:
The purpose in this test, is testing value to be empty string. But it set value for it.

Reply:
It sets the value as an empty string in the test below:

```
document.getElementById("value_can_be_empty_string").value = "";
```

Reply:
Why set empty below not set directly? I think it is not intuitive

Reply:
You may contact the author of this test and ask them this question. However I can venture a guess: the value of the `value` content attribute is the empty string at initialization, and setting it directly via the IDL setter ensures that we are testing specifically for the value sanitization algorithm.